rules/library.bzl

```diff
@@ -450,10 +524,58 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             tags = _MANUAL,
         )
         vendored_deps.append(import_name)
+        import_headers = native.glob(
+            ["**/*.h"],
+            exclude = ["**/.DS_Store"],
```

Comment:
blame MacOS

Reply:
this is important... otherwise we include deleted header files right... OMG 😱

Reply:
Let me circle back on this - I think the glob is terminated with `.h` so we shouldn't need an exclude! I think the previous code has an unterminated glob so they needed to exclude it